client/src/components/StudentCard.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import {
+  Typography,
+  Card,
+  CardContent,
+  Box,
+  Paper,
+} from "@mui/material";
+import { useState, useEffect } from "react";
+import styled from "@emotion/styled";
+import { DateTime } from "luxon";
+
+const StudentCard = () => {
+      const [cardData, setCardData] = useState([]);
+
+  // hook to fetch the event information and display it within the card
+  useEffect(() => {
+    fetch("http://localhost:3000/api/events/next")
+      .then(function (response) {
+        if (response.ok) {
+          return response.json();
+        }
+        throw `${response.status} ${response.statusText}`;
+      })
+      .then(function(data) {
+          console.log(typeof data[0].meeting_start);
+        setCardData(data);
+      })
+      .catch(function (error) {
+        console.log("An error occurred:", error);
+      });
+  }, []);
+
+  // function to convert timestamp string into an array of time an date
+  const timeSplitter=(element)=> {
+     return element.slice(0,16).split("T");
+  };
+  // typography styling
+  const TypographyInner = styled(Typography)({
+    variant: "h6",
+    display: "inline",
+    marginLeft: "10px",
+    color: "text.primary",
+    fontWeight: "bold",
+  });
+
+  return (
+		<Box>
+			<Paper>
+				<Typography variant="h5" sx={{ mb: 2, mt: 2 }} align="center">
+                    {/* optional heading, this can be changed */}
+					{cardData.length > 1 ? `You have ${cardData.length} Meetings`: "You have 1 Meeting"}
+				</Typography>
+			</Paper>
+
+			{cardData.map((element) => {
+				return (
+					<Card key={element.index} sx={{ mt: 2 }}>
+						<CardContent>
+							<Typography variant="h6">
```

Comment:
I think each of these need a key too. That should get rid of that error

Reply:
I will have a look at it.